ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/interface_ip_nat_keys.schema.yml

```diff
@@ -0,0 +1,119 @@
+type: dict
+$defs:
+  interface_ip_nat_static:
+    type: list
+    primary_key: original_ip
+    items:
+      type: dict
+      keys:
+        access_list:
+          type: str
+          description: "'access_list' and 'group' are mutual exclusive"
+        comment:
+          type: str
+        direction:
+          type: str
+          valid_values: [egress, ingress]
+          description: |-
+            Egress or ingress can be default. This depends on source/destination, EoS version and hardware platform.
+            EoS might remove this keyword in configuration. So, check configuration on targeted HW/SW .
+        group:
+          type: int
+          convert_types:
+            - str
+          min: 1
+          max: 65535
+          description: "'access_list' and 'group' are mutual exclusive"
+        original_ip:
+          type: str
+          description: IPv4 address
+        original_port:
+          type: int
+          convert_types:
+            - str
+          min: 1
+          max: 65535
+        priority:
+          type: int
+          convert_types:
+            - str
+          min: 0
+          max: 4294967295
+        protocol:
+          type: str
+          valid_values: [udp, tcp]
+        translated_ip:
+          type: str
+          required: true
+          description: IPv4 address
+        translated_port:
+          type: int
+          convert_types:
+            - str
+          min: 1
+          max: 65535
+          description: "requires 'original_port'"
+  interface_ip_nat:
+    type: dict
+    keys:
+      destination:
+        type: dict
+        keys:
+          dynamic:
+            type: list
+            primary_key: access_list
+            items:
+              type: dict
+              keys:
+                access_list:
+                  type: str
+                comment:
+                  type: str
+                pool_name:
+                  type: str
+                  required: true
+                priority:
+                  type: int
+                  convert_types:
+                    - str
+                  min: 0
+                  max: 4294967295
+          static:
+            $ref: "#/$defs/interface_ip_nat_static"
+      source:
+        type: dict
+        keys:
+          dynamic:
+            type: list
+            primary_key: access_list
+            items:
+              type: dict
+              keys:
+                access_list:
+                  type: str
+                comment:
+                  type: str
+                nat_type:
+                  type: str
+                  required: true
+                  valid_values:
+                    - overload
+                    - pool
+                    - pool-address-only
+                    - pool-full-cone
+                pool_name:
+                  type: str
+                  description: |-
```

Comment:
```suggestion
                  description: |
```

Reply:
@carlbuchmann why remove this? This just trims the trailing newline (I do the same in anything I add)

Reply:
Fixed in https://github.com/aristanetworks/ansible-avd/pull/2750/commits/11feb2f39043fcbee097a7d5b7caa793e9b85ed4